src/app/infobox/infobox.component.ts

```diff
@@ -47,7 +47,18 @@ export class InfoboxComponent implements OnInit {
       }
     });
   }
-
+  getQuery(q) {
```

Comment:
The issue was because the query parameters had only query and not the other fields.